udata_gouvfr/theme/templates/footer.html

```diff
@@ -75,11 +75,11 @@
 
             <section class="col-xs-9 col-xs-offset-3 col-sm-offset-0 col-sm-2 col-md-2 col-lg-2">
                 <a href="http://www.etalab.gouv.fr">
-                    <img class="logo" alt="logo"
+                    <img class="logo" alt="Etalab"
                         src="{{ theme_static('img/etalab-logo.png') }}"
                         width="152" height="50" />
-                    <p>2015 ETALAB</p>
-                </a>
+                </a><br />
+                <a href="https://github.com/opendatateam/udata/" title="{{ _('Open-source engine') }}">UData</a> <a href="https://udata.readthedocs.io/en/latest/changelog/" title="{{ _('Deployed version') }}">v{{ udata_version() }}</a>
```

Comment:
@jphnoel it comes from a recent addition in udata: https://github.com/opendatateam/udata/pull/593/files#diff-22a5c10e567d2b7e44d566765a43d278R28